packages/extension/src/utils/login.ts

```diff
@@ -0,0 +1,14 @@
+import { STORAGE_REMEMBER_SESSION } from '../constants';
+
+export const saveRememberSessionState = (checked: boolean) => {
+  localStorage.setItem(STORAGE_REMEMBER_SESSION, JSON.stringify(checked));
```

Comment:
Could you please store it in the chrome storage instead of local storage please?

Reply:
Done